### DISCUSSION TITLE
But why on github and not where sourcehut?

### DISCUSSION THREAD

**Discussion by chri2** [tinyBlogPub/microblog.pub#1] [General]
I'd say we should respect Thomas decision to mainly use sourcehut.

**Comment by nigini** [comment]
Hey, @chri2! Thanks for your suggestion. I mainly opted for GitHub because of its visibility, better tools, and the fact that other current users of the project use GitHub. (Also, I think the last time I looked into SourceHut, you couldn't do much with the free account.)

Finally, even Thomas had the project mirrored here, so I understood it as a personal preference (as the distributed nature of Git allows for). I will follow the crowd though. If you have a group that is working on this there, I'd happily work towards integrating our work.

Best.

**Reply by 1dom** [reply]
If tsileo/microblog.pub is abandoned, is it possible to take it over? If not, it would be good if the main repo here was at least marked as a fork of tsileos github repo. I can see it's got most of the commits, but it doesn't show up as a fork on the original which will make it harder for people to find. (Not sure if that's possible, or how).

**Reply by nigini** [reply]
I 100% agree! That was my intention, and this code is indeed a fork from Tsileo's. Maybe I messed it up by cloning all the involved repos in my machine and pushing only the final result here! :(

Thanks for pointing that out! (I guess it is still time to restart from a direct fork. Will work on this soon!)

**Reply by nigini** [reply]
Also, the sad part of Thomas going silent is that the project lost all the hard-built community and spaces, including the domain. 
It is what it is... let us do some reorganizing and feel where we want to go from here.

**Reply by 1dom** [reply]
I had a quick look at this, and it doesn't look like there's an easy way to do it other than deleting, forking, using the button and then pushing to that fork (https://stackoverflow.com/questions/5749246/how-do-i-change-which-github-project-i-forked-from)

I also noticed Thomas' servers seem to have come back online, so they might be active somewhere (e.g. https://microblog.pub is up again). I'd assume not here, but still worth a try: @tsileo? Any plans on picking up microblog.pub again?

**Reply by nigini** [reply]
OK. I have made the move and the main repo is now a direct fork! :)

**Comment by 3b0b** [comment]
Here's my (belated) take on it, which is mostly agreement with @nigini and @1dom, and seems to be what's happening already.

1. Yes, the free account on Sourcehut is quite limited.
2. While it's nice that the `git send-email` functionality Sourcehut uses is built into git itself, if the people interested in more rapid collaboration than is possible with the project owner currently absent from the Sourcehut project are more used to Github PRs, that's another obstacle to using Sourcehut.
3. He was at least maintaining a github fork. This suggests to me that he doesn't mind if there are forks on github. And now this actually is one, which is good.

If Thomas resumes activity and asks us to shut this down (or objects to the org name) I would hope for a polite resolution of those circumstances. In the meantime I don't think we should feel obligated to stick with his hosting platform of choice rather than the one where he kept a mirror.
